assets/js/blocks/cart-checkout/subscriptions-integration.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import {
+	ExperimentalOrderShippingPackages,
+	ShippingRatesControlPackage,
+} from '@woocommerce/blocks-checkout';
+import { registerPlugin } from '@wordpress/plugins';
+import { useStoreCart } from '@woocommerce/base-hooks';
+import { useMemo } from '@wordpress/element';
+
+const RenderSubscriptionPackages = () => {
+	const {
+		extensions: { subscriptions },
+	} = useStoreCart();
+
+	// Flatten all packages from recurring carts.
+	const packages = useMemo(
+		() =>
+			Object.values( subscriptions )
+				.map( ( recurringCart ) => recurringCart.shipping_rates )
+				.filter( Boolean )
+				.flat(),
+		[ subscriptions ]
+	);
+
+	return packages.map( ( { package_id: packageId, ...packageData } ) => (
+		<ExperimentalOrderShippingPackages key={ packageId }>
+			<SubscriptionPackage
+				key={ packageId }
+				packageId={ packageId }
+				packageData={ packageData }
+			/>
+		</ExperimentalOrderShippingPackages>
+	) );
+};
+
+const SubscriptionPackage = ( props ) => {
+	return <ShippingRatesControlPackage { ...props } />;
+};
```

Comment:
This seems to be useless now and I will delete it once I move the code if you don't mind Mike.